documentation-site/components/footer.js

```diff
@@ -65,32 +76,10 @@ export default () => (
       >
         Changelog
       </StyledLink>
+      <Link href="/blog">
+        <StyledLink>Blog</StyledLink>
```

Comment:
as is, the link is not tab-able - can we add the href prop to the styled link too?

Reply:
fixed, same with the `What's new in v8`